modules/claims_api/lib/claims_api/evidence_waiver_pdf/pdf.rb

```diff
@@ -52,7 +57,27 @@ def fill_pdf(response)
         page1_options(response),
         flatten: true
       )
+
       @page1_path = temp_path
     end
+
+    def storage_path
+      if Settings.evss.s3.uploads_enabled
+        "/tmp/#{SecureRandom.uuid}_final.pdf"
+      else
+        Rails.root.join('tmp',
+                        "#{SecureRandom.uuid}_final.pdf")
+      end
+    end
```

Comment:
curious if we still need this? it may be required but I'm not certain. the changes we added while troubleshooting were due to it adding the entire path from your home and then adding `/vets-api/public/etc`, but it's supposedly working without this on PoA? We should reevaluate this to make sure this IS required (and if it is, note that it's only for local dev to work).